### ISSUE TITLE
Not working with PoawerToys Mouse Jump.

### ISSUE THREAD

**Issue by chinook77** [mgth/LittleBigMouse#346]
Hi, is not working with PoawerToys Mouse Jump. The cursor stucks in the corner between monitors every time that activate the "jump".

**Comment by mgth** [comment]
You're right, its "by design" you'll have better luck with "corner crossing" algo but i's not perfect. I may introduce a "far distance" moving exception to prevent that.

**Comment by chinook77** [comment]
Thanks.

**Comment by mikeclayton** [comment]
I think what *maybe* happens is Mouse Jump moves the cursor but Little Big Mouse doesn't detect the move, and LBM bases its next move of the cursor on the last-known LBM location, not the actual current mouse location.

The PowerToys Mouse Crosshairs utility had a similar issue when Mouse Jump was first released - when Mouse Jump moved the cursors the Mouse Crosshairs utility didn't move its crosshairs to match - see https://github.com/microsoft/PowerToys/issues/24523 for details.

That was fixed by making PowerToys Mouse Jump call the Win API ```SendInput``` function to send a synthetic "mouse move" event which other applications (including Mouse Crosshairs) can detect and - in the case of Mouse Crosshairs it uses the event to  move the crosshair location.

See these links for the relevant bits of Mouse Jump that move the cursor and send the synthetic mouse move event:

* https://github.com/microsoft/PowerToys/blob/2189e7e1b9009c06f02692d6d4ab4b6d01fdd3bb/src/modules/MouseUtils/MouseJumpUI/Common/Helpers/MouseHelper.cs#L58
* https://github.com/microsoft/PowerToys/blob/2189e7e1b9009c06f02692d6d4ab4b6d01fdd3bb/src/modules/MouseUtils/MouseJumpUI/Common/Helpers/MouseHelper.cs#L104

You *might* be able to listen for that event in LBM and detect Mouse Jump moving the mouse from that, which might help when any other external utilities move the mouse as well...

**Comment by mgth** [comment]
Housekeeping pass: this report predates the full v5 rewrite of the mouse engine, and 5.3.0/5.4.x fixed a large batch of long-standing issues (stuck edges, silent startup failures, duplicated displays, sleep/resume, DPI handling…). This exact setup is unlikely to behave the same today, so I'm closing old reports in bulk.

If you still hit this on [5.4.1](https://github.com/mgth/LittleBigMouse/releases/tag/v5.4.1), a comment reopens it — fresh reports are always welcome.
